flytekit/models/literals.py

```diff
@@ -548,7 +549,7 @@ def from_flyte_idl(cls, pb2_object):
 
 
 class StructuredDatasetMetadata(_common.FlyteIdlEntity):
-    def __init__(self, format: str, structured_dataset_type: StructuredDatasetType):
+    def __init__(self, format: typing.Optional[str], structured_dataset_type: StructuredDatasetType = None):
```

Comment:
the type I'm not sure about, but the format shouldn't be optional.  We should always have a default, even if it's an "".

If we want to make structured_dataset_type optional, the `to_flyte_idl` and `from_flyte_idl` methods will also have to change to check for none-ness